eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -385,7 +385,8 @@ public function internalLoadContent($id, array $languages = null, $versionNo = n
         return $this->domainMapper->buildContentDomainObject(
             $spiContent,
             $this->repository->getContentTypeService()->loadContentType(
-                $spiContent->versionInfo->contentInfo->contentTypeId
+                $spiContent->versionInfo->contentInfo->contentTypeId,
+                $languages ?? []
```

Comment:
maybe we could extract those to variable, as under that we have that logic once more